packages/client/src/containers/RelatedItems/RelatedItems.Container.js

```diff
@@ -1,33 +1,74 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
 import PropTypes from 'prop-types';
 import ProductCard from '../../components/ProductCard/ProductCard.component';
 import { apiURL } from '../../apiURL';
+import { ViewBackCarousel } from '../../components/ViewCarousel/ViewBackCarousel/ViewBackCarousel';
+import { ViewAfterCarousel } from '../../components/ViewCarousel/ViewForwadCarousel/ViewAfterCarousel';
 import './RelatedItems.styles.css';
 
 export const RelatedItems = ({ category }) => {
   const [categoryProducts, setCategoryProducts] = useState([]);
+  const [offSet, setOffset] = useState(0);
+  const [dataFlag, setDataFlag] = useState(true);
 
-  useEffect(() => {
-    const fetchProductsByCategory = async () => {
-      const response = await fetch(`${apiURL()}/products?category=${category}`);
-      const categoryProductData = await response.json();
+  const fetchProductsByCategory = useCallback(async () => {
+    const response = await fetch(
+      `${apiURL()}/products?category=${category}&limit=3&offset=${offSet}`,
+    );
+    const categoryProductData = await response.json();
+
+    if (categoryProductData.data.length < 3) {
+      fetchAllProducts();
+    } else {
+      setDataFlag(true);
+      setCategoryProducts(categoryProductData.data);
+    }
+  }, [category, offSet]);
 
-      setCategoryProducts(categoryProductData);
-    };
+  const fetchAllProducts = async () => {
+    const response = await fetch(`${apiURL()}/products`);
+    const allProductsData = await response.json();
+    const productsData = allProductsData.slice(0, 3);
+    setCategoryProducts(productsData);
+    setDataFlag(false);
+  };
 
+  useEffect(() => {
     fetchProductsByCategory();
-  }, [category]);
+  }, [category, fetchProductsByCategory]);
+
+  const handleBackCarousel = () => {
+    if (offSet > 2) {
+      setOffset(offSet - 3);
+    }
+  };
+
+  const handleAfterCarousel = () => {
+    setOffset(offSet + 3);
+  };
 
   return (
     // TODO: will refactor function Number of items to display below. CLASS22-63"
     <div className="related-items">
       <h1 className="realtedItems-heading">Related Items</h1>
       <div className="related-items-display">
+        <div className="back-carousel">
+          <ViewBackCarousel text="" onClick={handleBackCarousel} />
```

Comment:
If it's not possible to go around backwards, then it would be better to disable this button if you're at the start of the items. Same for the next, if there is no more items, better to disable the button.